slimmer/slimmer.cpp

```diff
@@ -122,10 +122,20 @@ int parsed_main(int argc, char** argv) {
 
       //// PHOTONS ////
 
+      std::vector<std::pair<float, float>> em_directions; // Pairs of eta, phi for preselected leptons for cleaning
+
       for (auto& pho : event.photons) {
         output.n_pho_loose += pho.loose;
         output.n_pho_medium += pho.medium;
         output.n_pho_tight += pho.tight;
+        // if (pho.medium && pho.csafeVeto && pho.pixelVeto && pho.pt() > 25 && std::abs(pho.eta()) < 2.5)
+        if (pho.medium && pho.pt() > 175 && std::abs(pho.eta()) < 2.5 && pho.loose && pho.csafeVeto) {
```

Comment:
@DylanHsu I don't think we should require this photon pt to be so high before cleaning. (Though I wasn't cleaning at all before the sync)

Reply:
From what we know, the nominal analysis is not doing photon-jet cleaning at all. So we should either not do it at all or have the photon definition for cleaning be very tight. Have you found anything indicating that the other group does photon cleaning and if so what cuts do they use?